apps/src/sites/studio/pages/projects/featured.js

```diff
@@ -4,8 +4,16 @@ import ReactDOM from 'react-dom';
 import FeaturedProjects from '@cdo/apps/templates/projects/FeaturedProjects';
 
 $(document).ready(function () {
+  const script = document.querySelector('script[data-featuredprojects]');
+  const featuredProjectsData = JSON.parse(script.dataset.featuredprojects);
+  const currentFeaturedProjects = featuredProjectsData.currently_featured_projects;
+  const archiveUnfeaturedProjects = featuredProjectsData.archive_unfeatured_projects;
```

Comment:
The name `archiveUnfeaturedProjects` implies an action to archive unfeatured projects, which makes the rest of this code a bit confusing. I'd suggest something like `archivedFeaturedProjects`

Reply:
✅